canvas_sdk/handlers/simple_api/api.py

```diff
@@ -257,8 +260,9 @@ def __init_subclass__(cls, **kwargs: Any) -> None:
                     f"class attributes: {', '.join(f'{cls.__name__}.{name}' for name in names)}"
                 )
 
-    def _path_prefix(self) -> str:
-        return getattr(self, "PREFIX", None) or ""
+    @classmethod
+    def _path_prefix(cls) -> str:
+        return getattr(cls, "PREFIX", None) or ""
```

Comment:
```suggestion
        return getattr(cls, "PREFIX", "")
```

This is so pedantic I feel terrible 😂

Reply:
Don't worry, I am pro-pedantry. I'm not even sure exactly the code ended up like that.

Reply:
On second thought...these two statements are not actually the same (this change actually causes some tests to fail).

I need `PREFIX` to be a string because it gets prepended to the URL path. If a user sets `PREFIX` to `None`, then the final value will be `None` with this change, whereas without this change, the final value will be the empty string.

It's debateable whether setting it to `None` is valid usage; perhaps that should actually fail, however I think the developer intent in that case would be to say "My routes have no prefixes". So I think the argument here would be to be kind to the developer and accept it.

Perhaps reverting to what it was before and adding a comment is the way to go.

```python
        # getattr needs a default else it will raise an exception. We also need to
        # ensure that the final value is a string if the user specifies "None" as
        # as the prefix because this value gets prepended to the URL path
        return getattr(cls, "PREFIX", None) or ""
```